### DISCUSSION TITLE
Is there a way to access the individual elements of a glb in spoke?

### DISCUSSION THREAD

**Discussion by lukewhitingdev** [mozilla/hubs#3041] [General Questions & Discussion] [answered]
Just wondering since i would like to edit specify specific elements of my object to walkable to help with floor generation.

Like this (from blender)(selects individual object):
![blender](https://user-images.githubusercontent.com/43708922/93484089-68fa4a00-f8f9-11ea-8d9b-9d27e0c84962.JPG)

In spoke(selects what i feel is the rootNode and wont let me drill down):
![spoke](https://user-images.githubusercontent.com/43708922/93484371-bc6c9800-f8f9-11ea-86a5-01464a4faa12.JPG)

When on the object i can see that it recognises the individual parts since under the gltf info section i can see the different components and their vertex count etc.

**Comment by johnshaughnessy** [comment, marked as answer]
@lukewhitingdev Spoke does not currently "inflate" the nodes of a gltf/glb file when you load one into spoke, and instead treats the whole thing as one node. 

However, this sounds like a good [feature request in the spoke repo](https://github.com/mozilla/Spoke/issues?q=is%3Aissue+is%3Aopen+) because I believe this has been asked for before.
